build/bin/entrypoint

```diff
@@ -3,10 +3,6 @@
 # This is documented here:
 # https://docs.openshift.com/container-platform/3.11/creating_images/guidelines.html#openshift-specific-guidelines
 
-if ! whoami &>/dev/null; then
-  if [ -w /etc/passwd ]; then
-    echo "${USER_NAME:-externalsecret-operator}:x:$(id -u):$(id -g):${USER_NAME:-externalsecret-operator} user:${HOME}:/sbin/nologin" >> /etc/passwd
-  fi
-fi
+HOME=/home/${USER_NAME}
```

Comment:
this line is also in ` build/bin/user_setup`, is this correct?

Reply:
This line is indeed no longer needed.

Reply:
Fixed in https://github.com/ContainerSolutions/externalsecret-operator/pull/58/commits/59230810eb4a129f1983e319eff1efa118fec67e